### PR TITLE
Allow querying multiple channels at once in `GetChannelInformation`

### LINKED ISSUE #116: Allow querying multiple channels with `GetChannelInformation`
Twitch didn't document this properly in their [docs](https://dev.twitch.tv/docs/api/reference#get-channel-information), but it is possible to specify multiple `broadcaster_id` parameters (up to 100) and you will get back multiple objects for these, quite similarly to how it works in `GetClips` function.

I would need to fetch channel information for ~400 channels and doing that in bulks of 100 seems way more efficient than making separate requests for each of these, but I can't find a way to specify more than 1 `BroadcasterID` in `GetChannelInformationParams`.

I had an idea to change `GetChannelInformationParams.BroadcasterID string` to `GetChannelInformationParams.BroadcasterIDs []string` (also just like it's done in the `GetClips` function), but this would be a breaking change and existing applications would need to update, would you be okay with me PRing this change or should a different solution be considered instead?
Update to the above: I just decided to open a PR for that.

### PR DESCRIPTION
**Pull request by zneix**
This is a breaking change and would need to be addressed in the release changelog.

Closes #116

### CODE REVIEW

File: channels.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func (c *Client) SearchChannels(params *SearchChannelsParams) (*SearchChannelsRe
 }
 
 type GetChannelInformationParams struct {
-	BroadcasterID string `query:"broadcaster_id"`
+	BroadcasterIDs []string `query:"broadcaster_id"` // Limit 100
```
Comment (nicklaw5) [inline channels.go:55]: I'd prefer we not introduce this as a breaking change. Let's leave `BroadcasterID` as it is and introduce a new `BroadcasterIDs` field for handling this use case.

```suggestion
	// Deprecated: use BroadcasterIDs instead.
	BroadcasterID string `query:"broadcaster_id"`
	BroadcasterIDs []string `query:"broadcaster_id"` // Limit 100	
```

File: channels_test.go [hunk 2]
```diff
@@ -172,7 +172,7 @@ func TestGetChannelInformation(t *testing.T) {
 		c := newMockClient(testCase.options, newMockHandler(testCase.statusCode, testCase.respBody, nil))
 
 		resp, err := c.GetChannelInformation(&GetChannelInformationParams{
-			BroadcasterID: testCase.BroadcasterID,
+			BroadcasterIDs: []string{testCase.BroadcasterID},
```
Comment (nicklaw5) [inline channels_test.go:175]: Since we're keeping both struct fields, we need to have separate tests for both use cases.

Please revert this change and instead add a new test case to cater to `BroadcasterIDs`.

**Review by nicklaw5 — APPROVED** [review]

**Comment by zneix** [comment]
> I'd prefer we not introduce this as a breaking change. Let's leave `BroadcasterID` as it is and introduce a new `BroadcasterIDs` field for handling this new use case.

Did exactly this, added 2 new test cases that are using multiple BroadcasterIDs (one that "passes" and one that "fails").
